torch/utils/data/dataset.py

```diff
@@ -254,6 +254,31 @@ def __getitem__(self, index):
             return {k: dataset[index] for k, dataset in self.datasets.items()}
         return tuple(dataset[index] for dataset in self.datasets)
 
+    def __getitems__(self, indices: List) -> List[T_stack]:
```

Comment:
are the datasets in StackDataset supposed to have equal number of items?

Reply:
datasets should be the same by size, yes. good point.

Reply:
name stack has given as opposite to concatenate.
the behavior similar to torch.cat and torch.stack. 1st do flat tensor, 2nd creates new dim.